src/lib/operators/join_sort_merge.cpp

```diff
@@ -532,49 +550,55 @@ class JoinSortMerge::JoinSortMergeImpl : public AbstractReadOnlyOperatorImpl {
   **/
   void _join_cluster(const size_t cluster_id,
                      std::optional<MultiPredicateJoinEvaluator>& multi_predicate_join_evaluator) {
-    auto& left_cluster = (*_sorted_left_table)[cluster_id];
-    auto& right_cluster = (*_sorted_right_table)[cluster_id];
+    const auto& left_cluster = _sorted_left_table[cluster_id];
+    const auto& right_cluster = _sorted_right_table[cluster_id];
 
-    size_t left_run_start = 0;
-    size_t right_run_start = 0;
+    auto left_run_start = size_t{0};
+    auto right_run_start = size_t{0};
 
```

Comment:
Is there a reason for this change other than coding style guideline?

Reply:
Nope, just that we slowly move to `auto to stick`.